.github/workflows/build-test.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        libseccomp-version: [v2.5.0, v2.5.1, v2.5.3, v2.5.4, main]
+        libseccomp-version: [v2.5.5, v2.5.6, v2.6.0, main]
```

Comment:
Maybe we should keep running with `2.5.0` i.e. lowest supported version?

Reply:
@rusty-snake  thanks for the review.

okay, but should we do test only `v2.5.0` or `v2.5.0`, `v2.5.1`, `v2.5.3`, `v2.5.4` as before?

1. `[v2.5.0, v2.5.5, v2.5.6, v2.6.0, main]`
2. `[v2.5.0, v2.5.1, v2.5.3, v2.5.4, v2.5.5, v2.5.6, v2.6.0, main]`

Reply:
2. This would be a huge execution time.

I would go with 1. and even remove `2.5.5`. `2.5.1-6` only have added syscalls for newer kernels, some other adjustments for newer kernels and internal bugfixes, documentation.
As general rule: `main`, for each supported minor `2.x.0` and `2.x.max`.

Reply:
okay, thanks! fixed.